src/components/Header/SearchForm.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import OptionButton from '../Helper/OptionButton';
+
+const Wrapper = styled.div`
+  grid-area: 3 / 2 / 3 / 4;
+  display: flex;
+  flex-direction: column;
+`;
+
+const InputWrapper = styled.input`
+  font-size: 18px;
+  border: 0;
+  width: 780px;
+  height: 24px;
+  background: #fff;
+  padding: 6px;
+  border: 1px solid grey;
+  border-radius: 4px;
+`;
+
+const ButtonsWrapper = styled.div`
+  width: 800px;
+  display: flex;
+  margin-top: 10px;
+`;
+
+const Display = styled.p`
+  font-size: 18px;
+  margin: 5px;
+  padding: 5px;
+  color: white;
+`;
+
+const SearchButton = styled.button`
+  background-color: #dd1133;
+  height: 32px;
+  border: 0;
+  border-radius: 4px;
+  font-size: 14px;
+  font-weight: 600;
+  text-transform: uppercase;
+  margin: 4px 10px 4px 10px;
+  padding: 2px 10px 2px 10px;
+  margin-left: auto;
+`;
+
+class SearchForm extends Component {
+  state = {
+    searchValue: ''
+  };
+
+  handleInputChange = event => {
+    this.setState({ searchValue: event.target.value });
+  };
+
+  handleKeyPress = event => {
+    if (event.key === 'Enter') {
+      this.handleFormSubmit();
+    }
+  };
+
+  handleFormSubmit = event => {
+    event.preventDefault();
```

Comment:
Since you're using this function for button, probably there is no need in preventing default.